compiler-core/src/type_.rs

```diff
@@ -676,6 +693,7 @@ impl ValueConstructor {
     pub(crate) fn get_documentation(&self) -> Option<&str> {
         match &self.variant {
             ValueConstructorVariant::LocalVariable { .. } => Some("A locally defined variable."),
+            ValueConstructorVariant::LocalConstant { .. } => Some("A locally defined constant."),
```

Comment:
```suggestion
            ValueConstructorVariant::LocalConstant { .. } => Some("A locally defined variable."),
```

This is an implementation detail of the compiler, it's still a regular variable from the user's point of view.